doc/1/controllers/auth/check-token/snippets/check-token.cs

```diff
@@ -0,0 +1,15 @@
+try {
+  JObject response = (await kuzzle.Auth.LoginAsync("local", JObject.Parse("{username: 'foo', password: 'bar'}")));
```

Comment:
Why parenthesis around the call ?
Also you should wrap line greater than 80 column (apply everywhere)